addons/account/views/account_move_views.xml

```diff
@@ -308,6 +308,7 @@
                     <field name="name"/>
                     <field name="ref"/>
                     <field name="invoice_date"/>
+                    <field name="move_id" string="Journal Item Amount" filter_domain="[('move_id.amount_total', '=', self)]"/>
```

Comment:
Not really what i expected, look for the field debit and credit on the move lines